app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -686,8 +686,11 @@ export default {
           this.isATwilioWhatsAppChannel ||
           this.isAWhatsAppCloudChannel ||
           this.is360DialogWhatsAppChannel;
+        const isOnInstagram = this.isAInstagramChannel;
         if (isOnWhatsApp && !this.isPrivate) {
           this.sendMessageAsMultipleMessages(this.message);
+        } else if (isOnInstagram && !this.isPrivate) {
+          this.sendMessageAsSeparateMessages(this.message);
```

Comment:
Can't we `sendMessageAsMultipleMessages`

Reply:
In WhatsApp, we currently separate attachment messages only. We need to separate text messages in Instagram as well.